installing/installing-with-agent/installing-with-agent.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="installing-with-agent"]
+= Installing with Agent
+include::_attributes/common-attributes.adoc[]
+:context: installing-with-agent
+
+toc::[]
+
+== Agent installation
+Agent installation is based on the technology of Assisted Installer (AI), that is link:https://console.redhat.com[console.redhat.com] and the Infrastructure Operator. It is a command line tool that generates an image that runs the Assisted Service and the Assisted Installer Agent.
+//TODO>>include telemetry GA
+
+It is a subcommand that resides in the {product-title} installer and also utilizes the {product-title} installer for the input asset management.
+
+This subcommand generates a live {op-system-first} ISO with an embedded Ignition config containing `systemd` units to start the Assisted service, Agent, network, etc. that are running in containers pulled from the {product-title} release payload.
```

Comment:
It also contains a service that starts the cluster installation if all the required hosts have registered with assisted-service and passed validation.

Reply:
Could you please share the related jira?